src/databricks/labs/ucx/contexts/application.py

```diff
@@ -374,8 +374,23 @@ def table_ownership_grant_loader(self) -> TableOwnershipGrantLoader:
 
     @cached_property
     def pipelines_migrator(self) -> PipelinesMigrator:
+        include_pipeline_ids = (
+            self.named_parameters.get('include_pipeline_ids', '').split(',')
+            if 'include_pipeline_ids' in self.named_parameters
+            else None
+        )
```

Comment:
Kinda resolved already with the `get`

```suggestion
        include_pipeline_ids = self.named_parameters.get('include_pipeline_ids', '').split(',') or None
```

Same for `exclude_pipeline_ids` below

Reply:
I had this code before it resolves to an  empty list [''] instead of None so had to add an explicit else for None.